test/validators/movieList.test.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const Joi = require('joi');
+
+const MovieListValidator = require('../../lib/validators/movieList');
+
+describe('movie validator', () => {
```

Comment:
Same with this, to be more descriptive in the test output, you should change this to `movie list validator`.